src/types/route-requests.ts

```diff
@@ -26,3 +26,10 @@ export type AccessCodeDeleteRequest = {
   device_id?: string
   access_code_id: string
 }
+
+export type DeviceUpdateRequest = {
+  device_id: string
+  properties?: DeviceProperties
```

Comment:
Hmm this is tricky, the properties depend on the device type, maybe we could create a type BaseDeviceProperties and have LockProperties extend it